test/jdk/java/util/zip/ZipFile/CorruptedZipFiles.java

```diff
@@ -25,212 +25,238 @@
  * @bug 4770745 6218846 6218848 6237956
  * @summary test for correct detection and reporting of corrupted zip files
  * @author Martin Buchholz
+ * @run junit CorruptedZipFiles
  */
 
-import java.util.*;
-import java.util.zip.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
```

Comment:
I believe you should be using corresponding annotations from `org.junit.jupiter.api` instead.
From https://junit.org/junit5/docs/current/user-guide/index.html#migrating-from-junit4
> `@Before` and `@After` no longer exist; use `@BeforeEach` and `@AfterEach` instead.
> `@BeforeClass` and `@AfterClass` no longer exist; use `@BeforeAll` and `@AfterAll` instead.

Similarly the `@Test` annotation should be imported from `org.junit.jupiter.api`

Reply:
Noob question: Do you know where I find the jars for setting up junit5 in my IDE? jtreg's `junit-platform-console-standalone-1.9.2` does not seem to include these annotations.

Reply:
Seems my dev environment has `jtreg` < 7. Maybe that's a problem?

Reply:
Seems junit 5 support is a recent effort? 

https://mail.openjdk.org/pipermail/jdk-dev/2022-August/006869.html

Reply:
Scratch that. It seems my IDE was just not being very cooperative in suggesting or finding JUnit 5 API. When I wrote the imports my self it seems to have improved its behaviour.

Do you think it looks better now, @dfuch ?

Reply:
Yes - this a recent effort. I believe you will need at least jtreg 7, and JUnit 5.8.2

Reply:
It is worth noting that I did a new local build of jtreg and that may have updated the junit API jar. 

I would find that weird because the file name or version of the JAR file was not updated `jtreg/build/deps/junit/junit-platform-console-standalone-1.9.2.jar`. But maybe the content of the file changed?